catalog-rest-service/src/main/java/org/openmetadata/catalog/security/JwtFilter.java

```diff
@@ -87,7 +87,14 @@ public void filter(ContainerRequestContext requestContext) {
     } catch (RuntimeException runtimeException) {
       throw new AuthenticationException("Invalid token");
     }
-    String authorizedEmail = jwt.getClaim("email").as(TextNode.class).asText();
+    String authorizedEmail;
+    if (jwt.getClaim("email") != null) {
+       authorizedEmail = jwt.getClaim("email").as(TextNode.class).asText();
+    } else if (jwt.getClaim("sub") != null){
+      authorizedEmail = jwt.getClaim("sub").as(TextNode.class).asText();
+    } else {
+      throw new AuthenticationException("Email id and subject not present");
```

Comment:
Text should "Invalid JWT token, "email" or "subject" not present

Reply:
Ok, will change it.